contents/blog/collaborating-on-posthog.md

```diff
@@ -0,0 +1,47 @@
+---
+date: 2022-03-24
+title: Collaborating on PostHog
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+author: ["paolodamico"]
+featuredImage: ../images/blog/generic-release-notes.png
+featuredImageType: full
+categories: ["Product updates"]
+---
+
+PostHog is better accompanied. We quickly realized that as more users join an organization, the incremental value derived by each user increases. Consider for instance when you create a dashboard. You only create it once, but then with each user that joins your organization, the dashboard becomes more valuable. We found clear signals of this in our data: organizations with multiple teammates retain on average 30%+ better. We also recognized that conservatively we could increase insight discovery (our [main Product metric](https://posthog.com/handbook/product/metrics)) by more than 20% by improving collaboration workflows.
+
+
+Armed with the above we set out to make collaboration in PostHog as easily as possible, particularly gearing our efforts to making sure that larger teams are successful, with growing pains that tend to be negligible in small teams. We'll definitely continue improving on this, but we want to show you what we have today.
+
+
+## Introducing Collaboration
+
+### Use case one: Sharing
+
+- Permalinks
+- Granular dashboard permissions
+
+
+### Use case two: Discovering interesting stuff you weren't looking for
+- Project home
+- Default dashboard
+
+
+### Use case three: Leverage what others discovered
+- Saved insights + automatic insight naming
+- Connect to Data Management
+
+## What's next?
+Still in flux, depends on our learnings, ...
```

Comment:
I think we can refer to the roadmap items we have around sharing (from the offsite):
* Slack Previews
* Email subscriptions to dashboards and insights
* Embeddable graph images,